app/src/main/java/systems/v/coldwallet/Util/UIUtil.java

```diff
@@ -408,10 +408,13 @@ public static void setExecContractTx(final Activity activity, final Account send
         String time = new Timestamp(timestamp).toString();
         time = time.substring(0, time.indexOf("."));
         timestampTx.setText(time + "\n" + TimeZone.getDefault().getDisplayName());
-
+        String tempAttachment = attachment;
```

Comment:
`tempAttachment` is meaningless. Rename `tempAttachment` to `decodedAttachment`.